docs/080-breaking-changes.rst

```diff
@@ -86,12 +89,19 @@ New Restrictions
   This will make it safe to assume that the underlying type in the ABI is always ``uint8``.
 
 * Declarations with the name ``this``, ``super`` and ``_`` are disallowed, with the exception of
-  public functions and events.
+  public functions and events. The exception is to make it possible to declare interfaces of contracts
+  implemented in languages other than Solidity that do permit such function names.
 
 * Remove support for the ``\b``, ``\f``, and ``\v`` escape sequences in code.
   They can still be inserted via hexadecimal escapes, e.g. ``\x08``, ``\x0c``, and ``\x0b``, respectively.
 
 Interface Changes
 =================
 
-* Changed output of ``--combined-json``. JSON fields ``abi``, ``devdoc``, ``userdoc`` and ``storage-layout`` are sub-objects now. Before 0.8.0 they used to be serialised as strings.
+* The output of ``--combined-json`` has changed: JSON fields ``abi``, ``devdoc``, ``userdoc`` and
+  ``storage-layout`` are sub-objects now. Before 0.8.0 they used to be serialised as strings.
+
+* The "legacy AST" has been removed (``--ast-json`` on the commandline interface and ``legacyAST`` for standard-json).
```

Comment:
Are we really using "standard-json" as a term?  think we've used different style.

Reply:
We sometimes call it 'standard JSON interface', sometimes 'JSON-input-output interface".

Reply:
How about we introduce a new term (was thinking about this problem for a while, just never had an idea): "JSON Compiler Interface" or "JCI" for short. And avoid using the input-output and standard, etc.